### ISSUE TITLE
[InputForm.json]: User is unable to submit a form in which a name has an initial in it.

### ISSUE THREAD

**Issue by vagpt** [microsoft/AdaptiveCards-Mobile#106]
### Target Platforms

Android

### SDK Version

2023.12.12.1 (85454887)

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
OS: Android 13
Device: S20
Screen Reader: Talkback
AdaptiveCard Android App version: 2023.12.12.1 (85454887)

**Repro Steps:**

1. Open Adaptive Cards Visualizer Application.
2. Navigate to sample cards using tab key and activate it using Enter key.
3. A dropdown would appear. Activate it using Enter key.
4. Navigate to InputForms using tab key and activate it using Enter key.
5. Navigate to the Visual tab item present at the top. A sample card would appear.
6. Navigate to the Name edit box inside the sample card using tab key.
7. As observed the correct format is (Last, First). Try to enter a name with initials such as "Neha, G" a person whose name is G Neha.
8. Observe the issue.

**Actual Result:**
User is unable to submit a form in which the name has an initial in it. An error message pops up instructing the user to provide the right format.

**Expected Result:**
User should be able to submit a form in which the name has an initial in it.

**User Experience:**
All users would get affected if User is unable to submit a form in which the name has an initial in it.

Have feedback to share on Bugs ? please tag bug as “A11yRCA” and add your feedback in the comment section.

### Screenshots

https://github.com/microsoft/AdaptiveCards-Mobile/assets/95913748/cbe11e1a-cf29-4e38-942a-08b0fd541274

### Card JSON

```JSON
NA
```
